settings/Search.setting.php

```diff
@@ -213,7 +213,7 @@
     'group_name' => 'Search Preferences',
     'group' => 'Search Preferences',
     'name' => 'quicksearch_options',
-    'type' => 'String',
+    'type' => 'Array',
     'serialize' => CRM_Core_DAO::SERIALIZE_SEPARATOR_BOOKEND,
```

Comment:
@demeritcowboy perhaps this "serialize" doesn't belong here anymore?

Reply:
Oh maybe. Ok I'll check what happens without it.

Reply:
Yep seems good without it.